src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main_header">
+      <a
+        href="/"
+        class="logo_img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'Logo', a more descriptive text would be beneficial for SEO and accessibility.